### PR TITLE
Improvements to Halstack documentation site

### PR DESCRIPTION
**Pull request by GomezIvann**
**Checklist**
- [x] Build process is done without errors and all tests pass in the `/lib` directory.
- [x] Self-reviewed the code before submitting.
- [x] Meets accessibility standards.
- [x] Added/updated documentation to `/website` as needed.
- [x] Added/updated tests as needed.

**Description**
Summary of the updates made to Halstack's main site:

1. New GitHub link was added at the bottom of the sidenav of the repository.

image

2. New icons added for the edit and report links at the `DocFooter` component.

tempsnip

3. New UXPin section in /the introduction page, replacing the older section referencing Adobe XD assets.

image

### CODE REVIEW

File: website/screens/overview/introduction/IntroductionPage.tsx [hunk 1]
```diff
@@ -129,20 +129,50 @@ const sections = [
...
         content: (
           <DxcParagraph>
-            Implementing the previously described Design Guidelines so that
-            product designers can easily drag and drop the pre-created
-            components into their designs, obtaining automatic updates of these
-            components as the Design System evolves.
+            In the past, we used to rely on Adobe XD for our Design UI Kit. But
+            we recently decided to migrate all the assets to{" "}
+            <DxcLink href="https://www.uxpin.com/">UXPin</DxcLink> to start a
+            new and exciting journey. This decision brings, in addition to the
```
Comment (raquelarrojo) [inline website/screens/overview/introduction/IntroductionPage.tsx:143]: I would add `newWindow` to UXPin documentation link.

File: website/screens/overview/introduction/IntroductionPage.tsx [hunk 2]
```diff
@@ -129,20 +129,50 @@ const sections = [
...
+          },
+          {
+            title: "Prototyping Library",
+            content: (
+              <DxcParagraph>
+                The prototyping library is a Halstack CDK component-based design
+                library for fast prototyping, reducing the gap between designers
+                and developers. Also,{" "}
+                <DxcLink href="https://www.uxpin.com/merge">
+                  UXPin Merge
```
Comment (raquelarrojo) [inline website/screens/overview/introduction/IntroductionPage.tsx:168]: I would add `newWindow` to UXPin Merge documentation link.

**Review by raquelarrojo — CHANGES_REQUESTED** [review]

File: website/screens/overview/introduction/IntroductionPage.tsx (hunk 1 above)
Comment (GomezIvann) [reply website/screens/overview/introduction/IntroductionPage.tsx:143]: Agree!

File: website/screens/overview/introduction/IntroductionPage.tsx (hunk 2 above)
Comment (GomezIvann) [reply website/screens/overview/introduction/IntroductionPage.tsx:168]: Totally agree.

**Review by raquelarrojo — APPROVED** [review]

**Pull request merged**